financial-connections-example/src/main/java/com/stripe/android/financialconnections/example/FinancialConnectionsPlaygroundActivity.kt

```diff
@@ -182,21 +185,28 @@ class FinancialConnectionsPlaygroundActivity : AppCompatActivity() {
 
     private fun OpenForPaymentIntent.launchPaymentSheet(
         paymentSheet: PaymentSheet,
+        email: String,
     ) {
         PaymentConfiguration.init(
             context = this@FinancialConnectionsPlaygroundActivity,
             publishableKey = publishableKey
         )
+
+        val config = PaymentSheet.Configuration(
+            allowsDelayedPaymentMethods = true,
+            merchantDisplayName = "Example, Inc.",
+            customer = PaymentSheet.CustomerConfiguration(
+                id = requireNotNull(customerId),
+                ephemeralKeySecret = requireNotNull(ephemeralKey),
+            ),
+            defaultBillingDetails = PaymentSheet.BillingDetails(
+                email = email.takeIf { it.isNotBlank() },
+            ),
```

Comment:
Added this to make sure the customer email shows up in PaymentSheet’s email fields.